widgetsnbextension/setup.py

```diff
@@ -215,9 +215,7 @@ def run(self):
 
 setuptools_args = {}
 install_requires = setuptools_args['install_requires'] = [
-    'notebook>=4.2.0b1',
-    # TODO: Add ipywidgets dependency when ipywidgets does not depend
-    # on widgetsnbextension.
+    'notebook>=4.2.0',
 ]
```

Comment:
Removed comment: (widgetsnbextension should be installed on a notebook server where there is no kernel (eventually). It does not make sense to make it depend on ipywidgets)